### DISCUSSION TITLE
fc in phono3py format

### DISCUSSION THREAD

**Discussion by alinelena** [ttadano/alamode#193] [Q&A]
We try for various reasons but the main one is to check the calculations to generate fc constants using alm displacements and fc fitter but use them in both anphon and phono3py.

We did a simple test on Si using Tersoff d and got the hdf5 files using the python script along the lines from the manual.
with these fc we compute thermal conductivity in phono3py but the differences are quite big around 20-30% more than expected due to methodology.

if we do pure phono3py and pure alamode the resullts within some small error are the same.

I suspect the issue is the format in which the fc are dumped... any thoughts on this?

**Comment by ybC007** [comment]
Hello, I also want to use the fcs generated by alm in phono3py, because it's expensive to calculate fcs in phonopy. But I can't find a good method to achieve this target, may you provide some suggestions? Thanks alot.

**Comment by ttadano** [comment]
@ybC007 The python wrapper of [@ttadano/ALM](https://github.com/ttadano/ALM) may be useful for saving force constants in a hdf5 format of phono3py. Please also see the example script at https://github.com/ttadano/ALM/tree/develop/example/AlN.
